packages/salesforcedx-aura-language-server/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@salesforce/salesforcedx-aura-language-server",
+  "displayName": "Aura Language Server",
+  "version": "64.16.1",
```

Comment:
shouldn't this be same as latest extensions? so 65.9.0?

Reply:
it's ok either way. To the extension that needs them, they're just `*` and these aren't going to publish to npm anymore.